### ISSUE TITLE
twisted.trial.test.test_loader.ZipLoading tests need to run on Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8727]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8727|
|Type|enhancement|
|Created|2016-08-05 12:13:58Z|
|Branch|https://github.com/twisted/twisted/tree/8727-rodrigc-ziploading-py3|

On Python 3:

if I do:

python bin/trial twisted.trial.test.test_loader.Ziploading

the tests skip on Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8727 8727
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8727_rodrigc_ziploading_py3 8727-rodrigc-ziploading-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470399238271415 1470399238271415
changetime__1470719198285738 1470719198285738
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/466

**Comment by twisted-trac** [comment]
|[<img alt="markrwilliams's avatar" src="https://avatars.githubusercontent.com/u/704247?s=50" width="50" height="50">](https://github.com/markrwilliams)<a name="note_2"></a>|@markrwilliams commented|
|-|-|

Nice and simple!  Merge when the build is green!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 0c5a88ab36000416a057684e00f9c89b5ba3a5d4
```
#!CommitTicketReference repository="" revision="0c5a88ab36000416a057684e00f9c89b5ba3a5d4"
Merge pull request #466 from twisted/8727-rodrigc-ziploading-py3

Author: rodrigc
Reviewer: markrwilliams
Fixes: #8727

Allow twisted.trial.test.test_loader.ZipLoading tests to run on Python 3
```
